### PR TITLE
Add Tags into Event

### LINKED ISSUE #135: Add Tags to members and events

### PR DESCRIPTION
**Pull request by wr1159**
Closes #135

Add Tags as attribute in `Event.java`
* Changed Message Usage in CreateEventCommand
* Edited Relevant test files
* Display Tags in `Ui/EventCard.java`

image

### CODE REVIEW

File: src/main/java/seedu/ccacommander/logic/commands/CreateEventCommand.java [hunk 1]
```diff
@@ -22,11 +23,13 @@ public class CreateEventCommand extends Command {
             + "Parameters: "
             + PREFIX_NAME + "NAME "
             + PREFIX_LOCATION + "LOCATION "
-            + PREFIX_DATE + "DATE "
+            + PREFIX_DATE + "DATE \n"
             + "Example: " + COMMAND_WORD + " "
```
Comment (andrechuakj) [inline src/main/java/seedu/ccacommander/logic/commands/CreateEventCommand.java]: I think we can add the tag parameter here
```suggestion
            + PREFIX_DATE + "DATE "
            + "[" + PREFIX_TAG + "TAG]...\n"
            + "Example: " + COMMAND_WORD + " "
```

**Review by andrechuakj — CHANGES_REQUESTED** [review]
LGTM, just need to add in the tag parameter in the message usage.

File: src/main/java/seedu/ccacommander/logic/commands/CreateEventCommand.java (hunk 1 above)
Comment (wr1159) [reply src/main/java/seedu/ccacommander/logic/commands/CreateEventCommand.java]: Thanks!

**Review by andrechuakj — APPROVED** [review]
LGTM

**Review by SelwynAng — APPROVED** [review]
LGTM!

**Pull request merged**